components/elm/cime_config/testdefs/testmods_dirs/elm/polygonal_tundra/shell_commands

```diff
@@ -0,0 +1,9 @@
+./xmlchange LND_DOMAIN_FILE=domain.lnd.1x1pt_icycape_c240909.nc
+./xmlchange ATM_DOMAIN_FILE=domain.lnd.1x1pt_icycape_c240909.nc
+./xmlchange LND_DOMAIN_PATH="\$DIN_LOC_ROOT/share/domains/domain.clm"
+./xmlchange ATM_DOMAIN_PATH="\$DIN_LOC_ROOT/share/domains/domain.clm"
```

Comment:
If the `-res` for the test is changed, these `./xmlchange` won't be needed.

Reply:
Removed